### ISSUE TITLE
User returned from register endpoint actually is the same as the user passed

### ISSUE THREAD

**Issue by Ae-Mc** [fastapi-users/fastapi-users-db-sqlalchemy#7]
## Describe the bug

User create endpoint returns user that have been sent, not the user that was created at database.

## To Reproduce

Just add custom column to User database model and set it's server_default or default value.
When creating new user don't set that columns' value. Than in response this value will not have default value but sent value.

### Simple reproducable example.

Database User model:
```python
class User(Base, SQLAlchemyBaseUserTable):
    """Table for storing users"""

    created_at = Column(DateTime, default=datetime.now, nullable=False)
```

Pydantic User schemas:
```python
class User(models.BaseUser):
    """User fetch pydantic model"""

    created_at: datetime | None  # None is to create users without Pydantic's validation exception


class UserCreate(models.BaseUserCreate):
    """User's creation pydantic scheme"""


class UserUpdate(models.BaseUserUpdate):
    """User's update pydantic scheme"""


class UserDB(User, models.BaseUserDB):
    """User stored in DB pydantic model"""
```

Than add registration router and test.

### Request and response examples
Request:
```json
{
  "email": "some-email@some-server.com",
  "password": "some-password",
  "is_active": true,
  "is_superuser": true,
  "is_verified": true
}
```
Response:
```json
{
  "id": "ecdd896e-2c76-42ba-b374-d5281e588e87",
  "email": "some-email@some-server.com",
  "is_active": true,
  "is_superuser": false,
  "is_verified": false,
  "created_at": null
}
```

## Expected behavior

Successful user creation with automatically set created_at returned from registration endpoint.

For above request expected response is:
```json
{
  "id": "ecdd896e-2c76-42ba-b374-d5281e588e87",
  "email": "some-email@some-server.com",
  "is_active": true,
  "is_superuser": false,
  "is_verified": false,
  "created_at": "2022-01-16T00:16:22.626191"
}
```

## Configuration
 - Python version : 3.10.1
 - FastAPI version : 0.71.0
 - FastAPI Users version : 9.2.2

### FastAPI Users configuration

Standard configuration from docs

**Comment by Ae-Mc** [comment]
The easiest way to fix is to replace this line:
https://github.com/fastapi-users/fastapi-users-db-sqlalchemy/blob/afecb6b3380d52cdc4b21743f45a64c065acd559/fastapi_users_db_sqlalchemy/__init__.py#L117
with something like this:
```python
    return await self.get(user.id)
```
to fetch user with already set properties. The same is made by ormar adapter.

**Comment by frankie567** [comment]
Yes, I agree that it will be better like this.

**Comment by frankie567** [comment]
Fixed in e24d824ed77a14cc7e856cccf811aa7789833ad7
